tests/src/test/java/io/streamnative/pulsar/handlers/kop/PulsarAuthEnabledKafkaTest.java

```diff
@@ -13,10 +13,23 @@
  */
 package io.streamnative.pulsar.handlers.kop;
 
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
 /**
  * {@link PulsarAuthEnabledTestBase} with `entryFormat=kafka`.
  */
 public class PulsarAuthEnabledKafkaTest extends PulsarAuthEnabledTestBase {
+    @BeforeClass
+    @Override
+    protected void setup() throws Exception {
+        super.setup();
+    }
+    @AfterClass
+    @Override
+    protected void cleanup() throws Exception {
+        super.cleanup();
+    }
```

Comment:
I think these lines could be removed, as well as `PulsarAuthEnabledPulsarTest`. The fix should be adding `@AfterClass` annotation to `PulsarAuthEnabledTestBase#cleanup`.